.github/workflows/e2e_android.yml

```diff
@@ -36,19 +36,20 @@ env:
   # Filepath for firebase distribution key. Also used by fastlane
   FIREBASE_DIST_KEY_FILEPATH: ${{ secrets.FIREBASE_DIST_KEY_FILEPATH }}
   # Slack API token used by fastlane
-  # SLACK_API_TOKEN: ${{ secrets.SLACK_API_TOKEN }}
+  SLACK_API_TOKEN: ${{ secrets.SLACK_API_TOKEN }}
 
 jobs:
-  # start_slack_thread:
-  #   name: Start Slack thread
-  #   uses: ./.github/workflows/start_slack_thread.yml
-  #   secrets: inherit
-  #   with:
-  #     channel_name: va-mobile-build-alerts
-  #     message: 'E2E Android build process starting. Please see :thread: for results. This process may take a while.'
+  start_slack_thread:
+    if: github.event_name == 'schedule'
+    name: Start Slack thread
+    uses: ./.github/workflows/start_slack_thread.yml
+    secrets: inherit
+    with:
+      channel_name: va-mobile-build-alerts
+      message: 'E2E Android build process starting. Please see :thread: for results. This process may take a while.'
```

Comment:
I think including something in the message about running tests would make the intent of the thread more clear, as it doesn't mention anything about actually running the tests. Maybe something like
```
Starting E2E Android tests. Please see :thread: for results. This process may take a while.
```

That also brings up another small concern that's beyond the scope of this ticket. I noticed this is being sent to the `va-mobile-build-alerts` channel. This channel has been primarily used for build alerts, as suggested in the channel name. If we're going to start adding different types of alerts to this channel, I wonder if it'd make sense to rename the channel to represent a broader range of alerts. Since E2E tests are still somewhat related to builds (i.e. they tell us when something would be wrong with a build), maybe they still belong with build alerts, but just something for us to think about moving forward

Reply:
updated, I see the point of renaming the channel at some point, but at the moment it still has a descriptive enough name